doc/user/inputs/settings.rst

```diff
@@ -97,3 +97,39 @@ Detail assembly numbers
 
 Detail all assemblies
     The ``detailAllAssems`` setting makes all assemblies in the problem detail assemblies
+
+.. _kinetics-settings:
+
+Kinetics settings
+-----------------
+In reactor physics analyses it is standard practice to represent reactivity
+in either absolute units (i.e., dk/kk' or pcm) or in dollars or cents. To
+support this functionality, the framework supplies the ``beta`` and
+``decayConstants`` settings to apply the delayed neutron fraction and
+precursor decay constants to the Core parameters during initialization.
+
+These settings come with a few caveats:
+
+    1. The ``beta`` setting supports two different meanings depending on
+       the type that is provided. If a float is given, then this setting
+       is interpreted as the effective delayed neutron fraction for the
+       system and is applied to the Core's ``beta`` parameter. If a list
```

Comment:
Discussing the ``beta`` parameter directly might be TMI in this context.

Reply:
That's true, but I want to make it clear how the setting is being applied versus just saying what it means. Is this just confusing to a new reader or is there a better place to put this type of information? I guess I could have this in a docstring too and then link directly to it.